### PR TITLE
Add nfdi4ing/profiles as PID for metadata application profiles

### PR DESCRIPTION
**Pull request by MattGroenewald**
Metadata application profiles created by the NFDI4Ing Metadata Profile Service (MPS) will have the namespace https://w3id.org/nfdi4ing/profiles/ followed by a UUID. This pull request shall provide the referral to open profiles in the designated MPS editor view. Later commits will add functionality for direct request of .ttl, .json among others via http accept statements.

### CODE REVIEW

**Comment by dgarijo** [comment]
@MattGroenewald please seek approval from @JohannesLipp, @doigl or @Zack-83, the maintainers of this w3id.

File: nfdi4ing/profiles/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Technical University of Darmstadt   
+[University and State Library Darmstadt](https://ulb.tu-darmstadt.de)   
+Darmstadt, Germany   
+ORCID: [0000-0002-9826-018X](https://orcid.org/0000-0002-9826-018X)
+
+**TBA**  
+XXXXXX
+xxx-xxx
+xxx-xxx-xxx
+City
```
Comment (Zack-83) [inline nfdi4ing/profiles/README.md:24]: who should be added here?

**Review by Zack-83 — APPROVED** [review]

**Comment by dgarijo** [comment]
I will wait until the comment by @Zack-83 gets resolved

File: nfdi4ing/profiles/README.md (hunk 1 above)
Comment (mfuhrmans) [reply nfdi4ing/profiles/README.md:24]: @Marius: please also include your ORCID (currently line 25).

Comment (MattGroenewald) [reply nfdi4ing/profiles/README.md]: I updated the contact details.

File: nfdi4ing/profiles/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+[University and State Library Darmstadt](https://ulb.tu-darmstadt.de)   
+Darmstadt, Germany   
+ORCID: [0000-0002-9826-018X](https://orcid.org/0000-0002-9826-018X)
+
+**Marius Politze**  
+Research Process & Data Management
+RWTH Aachen University
+[IT Center](https://www.itc.rwth-aachen.de/)
+Aachen, Germany
+ORCID: [0000-0003-3175-0659](https://orcid.org/0000-0003-3175-0659)
```
Comment (TallTed) [inline nfdi4ing/profiles/README.md:25]: to preserve line breaks
```suggestion
**Marius Politze**  
Research Process & Data Management  
RWTH Aachen University  
[IT Center](https://www.itc.rwth-aachen.de/)  
Aachen, Germany  
ORCID: [0000-0003-3175-0659](https://orcid.org/0000-0003-3175-0659)
```

File: nfdi4ing/profiles/README.md (hunk 1 above)
Comment (mpolitze) [reply nfdi4ing/profiles/README.md]: @mfuhrmans, @Zack-83 approved by me. thanks to @MattGroenewald, contact details look good.

**Pull request merged**